yammbs/_store.py

```diff
@@ -476,21 +476,15 @@ def from_cached_result_collection(
 
         return store
 
-    def optimize_mm(
-        self,
-        force_field: str,
-        n_processes: int = 2,
-        chunksize=32,
-    ):
-        from yammbs._minimize import _minimize_blob
-
+    def _map_inchi_keys_to_qm_conformers(self, force_field: str) -> dict[str, list]:
         inchi_keys = self.get_inchi_keys()
 
-        inchi_key_qm_conformer_mapping = defaultdict(list)
+        mapping = defaultdict(list)
 
         for inchi_key in inchi_keys:
             molecule_id = self.get_molecule_id_by_inchi_key(inchi_key)
 
+            # TODO: Should the session be inside or outside of the inchi loop?
```

Comment:
As written, I think the session needs to be here because `get_molecule_id_by_inchi_key` also locks the db. If the code inside `get_molecule_id_by_inchi_key` were in-lined, the session could probably be moved outside the loop.